From dbf3e30cc1e77823e8da29bf8e3190b6d031c062 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Mon, 17 Jun 2013 17:35:20 +0300
Subject: [PATCH 0399/1089] dpaa_eth: print probed interfaces

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Print a message for each probed interface.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
Change-Id: If108fc8828bc03c44b1e539eb340eb086cd4b2c7
Reviewed-on: http://git.am.freescale.net:8181/3092
Reviewed-by: Sovaiala Cristian-Constantin-B39531 <Cristian.Sovaiala@freescale.com>
Reviewed-by: Radulescu Ruxandra Ioana-B05472 <ruxandra.radulescu@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth.c         | 2 ++
 drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c | 2 ++
 drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c  | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
index 17c8ac4..a534a30 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
@@ -3227,6 +3227,8 @@ dpaa_eth_priv_probe(struct platform_device *_of_dev)
 
 	dpaa_eth_sysfs_init(&net_dev->dev);
 
+	printk(KERN_INFO "fsl_dpa: Probed interface %s\n", net_dev->name);
+
 #ifdef CONFIG_FSL_DPAA_ETH_UNIT_TESTS
 	/* The unit test is designed to test private interfaces */
 	dpa_unit_tests(net_dev);
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
index 43e119d..d4610c9 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
@@ -388,6 +388,8 @@ static int dpaa_eth_macless_probe(struct platform_device *_of_dev)
 
 	dpaa_eth_sysfs_init(&net_dev->dev);
 
+	printk(KERN_INFO "fsl_dpa_macless: Probed macless interface %s\n", net_dev->name);
+
 	return 0;
 
 netdev_init_failed:
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
index 2b75b1e..2d06be1 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
@@ -750,6 +750,8 @@ dpaa_eth_shared_probe(struct platform_device *_of_dev)
 
 	dpaa_eth_sysfs_init(&net_dev->dev);
 
+	printk(KERN_INFO "fsl_dpa_shared: Probed shared interface %s\n", net_dev->name);
+
 	return 0;
 
 netdev_init_failed:
-- 
2.0.2

